posts/urls.py

```diff
@@ -3,8 +3,9 @@
 
 app_name = 'posts'
 urlpatterns = [
-    path('new', views.CreatePostView.as_view(), name='new'),
     path('<int:pk>/edit', views.EditPostView.as_view(), name='edit'),
     path('<int:pk>/comments/new', views.CreateCommentView.as_view(), name='new-comment'),
     path('<int:pk>', views.PostDetailView.as_view(), name='detail'),
+    path('new', views.CreatePostView.as_view(), name='new'),
+    path('', views.MyPostsView.as_view(), name='my-posts'),
```

Comment:
so this would be 'localhost:8000/posts/' correct?

Reply:
Yes

Reply:
For now at least, I'm planning to move this under the user's profile